code/modules/hydroponics/grown.dm

```diff
@@ -80,14 +80,9 @@
 
 /obj/item/food/grown/attackby(obj/item/O, mob/user, params)
 	..()
-	if (istype(O, /obj/item/plant_analyzer))
-		var/obj/item/plant_analyzer/plant_analyzer = O
-		to_chat(user, plant_analyzer.scan_plant(src))
-	else
-		if(seed)
-			for(var/datum/plant_gene/trait/T in seed.genes)
-				T.on_attackby(src, O, user)
-
+	if(seed)
+		for(var/datum/plant_gene/trait/T in seed.genes)
+			T.on_attackby(src, O, user)
```

Comment:
```suggestion
		for(var/datum/plant_gene/trait/trait as anything in seed.genes)
			trait.on_attackby(src, O, user)
```

Reply:
This is getting removed entirely in this refactor #56841 , should I still make this change?

Reply:
ehhhhhhhh leave this as a draft until that ones merged then